gobblin-metastore/src/main/java/gobblin/metastore/util/DatabaseJobHistoryStoreSchemaManager.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.metastore.util;
+
+import com.google.common.io.Closer;
+import org.apache.commons.configuration.AbstractConfiguration;
+import org.apache.commons.configuration.CompositeConfiguration;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.commons.configuration.SystemConfiguration;
+import org.apache.commons.lang.StringUtils;
+import org.flywaydb.core.api.FlywayException;
+import org.flywaydb.core.api.MigrationInfoService;
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.internal.info.MigrationInfoDumper;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+import gobblin.metastore.DatabaseJobHistoryStore;
+
+
+/**
+ * A utility class for managing the {@link DatabaseJobHistoryStore} schema.
+ *
+ * @author Joel Baranick
+ */
+public class DatabaseJobHistoryStoreSchemaManager implements Closeable {
+
+  private final Flyway flyway;
+
+  private DatabaseJobHistoryStoreSchemaManager(Properties properties) {
+    flyway = new Flyway();
+    flyway.configure(properties);
+    flyway.setClassLoader(this.getClass().getClassLoader());
+  }
+
+  public static DataSourceBuilder builder() {
+      return new Builder();
+  }
+
+  private static FinalBuilder builder(Properties properties) {
+    return new Builder(properties);
+  }
+
+  public void migrate() throws FlywayException {
+    flyway.migrate();
+  }
+
+  public void info() throws FlywayException {
+    MigrationInfoService info = flyway.info();
+    System.out.println(MigrationInfoDumper.dumpToAsciiTable(info.all()));
+  }
+
+  @Override
+  public void close() throws IOException {
+  }
+
+  public static void main(String[] args) throws IOException {
+    if (args.length < 1 || args.length > 2) {
+      printUsage();
+    }
+    Closer closer = Closer.create();
+    try {
+      CompositeConfiguration config = new CompositeConfiguration();
```

Comment:
I haven't used Commons CLI.  I would rather leave it as it is.  I don't have the time to change this now.